client/sites-dashboard/index.ts

```diff
@@ -1,13 +1,22 @@
 import { isEnabled } from '@automattic/calypso-config';
+import { get } from 'lodash';
 import page from 'page';
 import { makeLayout, render as clientRender } from 'calypso/controller';
+import { sites, siteSelection } from 'calypso/my-sites/controller';
+import { getSiteBySlug, getSiteHomeUrl } from 'calypso/state/sites/selectors';
 import { sitesDashboard } from './controller';
 
 export default function () {
+	page( '/sites/:site', ( context ) => {
```

Comment:
As we can't use feature flags in `sections.js`, I took over the `my-sites` routing responsibilities in the Sites Dashboard router. Is it good approach?

Reply:
> Is it good approach?

Yeah it looks like we're going to need to handle the feature flag logic inside these routing functions since we can't put it in `sections.js`.

Rather than duplicate the code we could import `client/my-sites/index.js` and if the feature flag is disabled then we call its main routing function?

Another option would be to leave the `/sites` and `/sites-dashboard` sections as they are, and then have each redirect to the other one depending on the feature flag state.

Reply:
>Another option would be to leave the /sites and /sites-dashboard sections as they are, and then have each redirect to the other one depending on the feature flag state.

I really prefer that. It's behind a feature flag, so it makes sense to change as little as possible.